src/components/composer/grid_composer/grid_composer.ts

```diff
@@ -184,7 +184,11 @@ export class GridComposer extends Component<Props, SpreadsheetChildEnv> {
   }
 
   private updateCellReferenceVisibility() {
```

Comment:
By curiosity, why do we store `isCellReferenceVisible` and call a function at each updateProps to update it, instead of computing it on the fly at each render ?